definitions/checks/foreman/check_https_proxies.rb

```diff
@@ -0,0 +1,34 @@
+module Checks
+  class CheckHttpsProxies < ForemanMaintain::Check
+    metadata do
+      label :https_proxies
+      for_feature :foreman_database
+      description 'Check for HTTPS proxies from the database'
+      manual_detection
+    end
+
+    def run
+      https_proxies = find_https_proxies
+      unless https_proxies.empty?
+        https_proxy_names = https_proxies.map { |proxy| proxy['name'] }
+        question = "Syncing repositories through an \'HTTP Proxy\' that uses the https:// "\
```

Comment:
You shouldn't need to escape the single quotes here?

Also, I'd say HTTPS protocol, not https:// (that's more of an URI format)